docs/02-lazy-services.md

```diff
@@ -0,0 +1,2 @@
+## Lazy services
+To allow for publishers to be lazy loaded you can use [Symfony's proxy manager bridge](http://symfony.com/doc/current/components/dependency_injection/lazy_services.html).
```

Comment:
It doesn't break if you don't use it. It is just less optimal in some use cases.
> If you don't install the ProxyManager bridge, the container will just skip over the lazy flag and simply instantiate the service as it would normally do.